src/analysis/max_min_driving_force.jl

```diff
@@ -126,13 +165,20 @@ function max_min_driving_force(
         mod(model, opt_model)
     end
 
+    @constraint(opt_model, minDF .<= -dgrs[active_ridxs] .* sign.(fluxes[active_ridxs]))
+
+    @objective(opt_model, Max, minDF)
+
     optimize!(opt_model)
 
     is_solved(opt_model) || return nothing
 
     return (
         mmdf = objective_value(opt_model),
-        dgs = Dict(rid => value(dgs[i]) for (i, rid) in enumerate(reactions(model))),
-        cs = Dict(mid => exp(value(logcs[i])) for (i, mid) in enumerate(mids)),
+        dgrs = Dict(rid => value(dgrs[i]) for (i, rid) in enumerate(reactions(model))),
+        cs = Dict(
```

Comment:
I guess we can spell out full concentrations here, 'cs' is superoverloaded already :D